packages/arco-lib/src/generated/types/Table.ts

```diff
@@ -32,6 +32,13 @@ export const ColumnSchema = Type.Object({
     category: Category.Basic,
     description: 'The text you want to display instead of raw text.',
   }),
+  width: Type.Optional(Type.Number({
+    title: 'Width',
+  })),
+  ellipsis:Type.Optional(Type.Boolean({
```

Comment:
What is this property for?

Reply:
If the cell content exceeds the length, it is automatically omitted and displays `...`    
I'll add a description

Reply:
But the code seems like if not `ellipsis`, the text will be hidden?
```
colItem = <span title={column.ellipsis ? text : ''}>{text}</span>;
```

Reply:
Only the title will be hidden, it will be shown when ellipsis is on and hovering